airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/ctid/CtidUtils.java

```diff
@@ -53,43 +60,43 @@ public static StreamsCategorised categoriseStreams(final StateManager stateManag
         }
 
         if (streamState.has("state_type")) {
+          final AirbyteStreamNameNamespacePair pair = new AirbyteStreamNameNamespacePair(streamDescriptor.getName(),
+                                                                                         streamDescriptor.getNamespace());
           if (streamState.get("state_type").asText().equalsIgnoreCase("ctid")) {
+            System.out.println("IN CTID ");
             statesFromCtidSync.add(stateMessage);
-            streamsStillInCtidSync.add(new AirbyteStreamNameNamespacePair(streamDescriptor.getName(), streamDescriptor.getNamespace()));
+            stillInCtidStreamPairs.add(pair);
           } else if (streamState.get("state_type").asText().equalsIgnoreCase("xmin")) {
             if (shouldPerformFullSync(currentXminStatus, streamState)) {
-              final AirbyteStreamNameNamespacePair pair = new AirbyteStreamNameNamespacePair(streamDescriptor.getName(),
-                  streamDescriptor.getNamespace());
               LOGGER.info("Detected multiple wraparounds. Will perform a full sync for {}", pair);
-              streamsStillInCtidSync.add(pair);
+              stillInCtidStreamPairs.add(pair);
             } else {
+              xminSyncStreamPairs.add(pair);
               statesFromXminSync.add(stateMessage);
             }
+          } else if (streamState.get("state_type").asText().equalsIgnoreCase("standard")) {
+            standardSyncStreamPairs.add(pair);
+            statesFromStandardSync.add(stateMessage);
           } else {
             throw new RuntimeException("Unknown state type: " + streamState.get("state_type").asText());
```

Comment:
The current states saved in the DB dont have the state_type attributes for standard sync. This would result in them breaking.